src/app/home/home.component.ts

```diff
@@ -10,6 +10,8 @@ import { Router } from '@angular/router';
 })
 export class HomeComponent implements OnInit {
   sidenavOpened = true;
+  altViewOpened = false;
```

Comment:
you should only need one boolean for this for now (since we're just using two views). the logic check can just be 
ngIf altViewIsOpened and ngIf !altViewIsOpened